tools/reviewer/web/src/app/dashboard/reviews-panel/review/editable-property/editable-property.component.spec.ts

```diff
@@ -1,20 +1,20 @@
 import { async, ComponentFixture, TestBed } from '@angular/core/testing';
 
-import { BugComponent } from './bug.component';
+import { EditablePropertyComponent } from './editable-property.component';
 
-describe('BugComponent', () => {
-  let component: BugComponent;
-  let fixture: ComponentFixture<BugComponent>;
+describe('EditablePropertyComponent', () => {
+  let component: EditablePropertyComponent;
+  let fixture: ComponentFixture<EditablePropertyComponent>;
 
   beforeEach(async(() => {
     TestBed.configureTestingModule({
-      declarations: [ BugComponent ]
+      declarations: [ EditablePropertyComponent ]
     })
     .compileComponents();
   }));
 
   beforeEach(() => {
-    fixture = TestBed.createComponent(BugComponent);
+    fixture = TestBed.createComponent(EditablePropertyComponent);
```

Comment:
Do you think there's any reason to have unit tests of components?
Thing is, they are usually huge because you have to import all angular interface modules there. You have to import them inside your app module, and inside all your spec files too. It makes code less DRY, that's why it becomes more and more difficult to maintain the unit tests. But what's more important, the tests don't make much sense. All complex logic should be inside services (local and global) and if a component has an error, you already get a message in your console.

Please, let me know your thoughts about it.

Reply:
Yeah, I haven't seen component tests doing much good other than just loading and saying all is good.